[chart/redis-ha] split-brain-fix.sh is executed on "sh" but uses "==" instead of "=" for comparison

Hello,
I'm trying to install `argocd` helm chart and it has a dependency on `redis-ha`.

Observed that sometimes, the redis pods keep crashing and go into crash loop back off state; and on checking logs for errors I noticed the following error appearing repeatedly in the split-brain-fix container logs

```
+ [ 10.104.191.33 == 10.104.191.33 ]                                                                                                                      │
/readonly-config/fix-split-brain.sh: 276: [: 10.104.191.33: unexpected operator
```
Note: I edited the container command/args to run the script in debug mode by adding -x

Though I am not sure if the redis container crash loop is related to this error, while debugging I found that this error is caused because the script is invoked using "sh" which does not support comparison using "==" operator
Comparison using "==" is being done in [fix-split-brain.sh:276](https://github.com/DandyDeveloper/charts/blob/master/charts/redis-ha/templates/_configs.tpl#L468)

I think this issue is causing the flow-of-control to always go into the else block

Reference: https://www.shellcheck.net/wiki/SC3014

--- a/charts/redis-ha/Chart.yaml
+++ b/charts/redis-ha/Chart.yaml
@@ -5,7 +5,7 @@ keywords:
 - redis
 - keyvalue
 - database
-version: 4.22.1
+version: 4.22.2
 appVersion: 7.0.4
 description: This Helm chart provides a highly available Redis implementation with a master/slave configuration and uses Sentinel sidecars for failover management
 icon: https://upload.wikimedia.org/wikipedia/en/thumb/6/6b/Redis_Logo.svg/1200px-Redis_Logo.svg.png

--- a/charts/redis-ha/templates/_configs.tpl
+++ b/charts/redis-ha/templates/_configs.tpl
@@ -465,7 +465,7 @@
         # where is redis master
         identify_master
 
-        if [ "$MASTER" == "$ANNOUNCE_IP" ]; then
+        if [ "$MASTER" = "$ANNOUNCE_IP" ]; then
             redis_role
             if [ "$ROLE" != "master" ]; then
                 reinit

